### DISCUSSION TITLE
NZP stuck on main menu [PS VITA]

### DISCUSSION THREAD

**Discussion by adrianotwink** [nzp-team/nzportable#644] [General]
None of the controls work, I can get past the screen that asks if I want to check for updates or press start to go to the main menu but the moment I get to the main menu I can't do anything. It's like frozen or stuck at the main menu. Help please.

**Comment by adrianotwink** [comment]
https://github.com/nzp-team/nzportable/assets/158851936/a05bdc01-5137-46cc-b920-2925ec5721f2

**Comment by PSRYT** [comment]
Your Vita froze. Safe mode; Restart; You should be fine.

**Reply by adrianotwink** [reply]
It was not my vita freezing. My vita was able to perform other tasks at hand without the need to restart. Plus restarting didn't fix it either. I had to uninstall a plugin that for some reason kept the game from working.

**Reply by BCDeshiG** [reply]
What plugin did you uninstall?
